sftp/sftp_test.go

```diff
@@ -138,10 +138,11 @@ func TestUpload(t *testing.T) {
 
 	mockSFTPClient := mock_sftp.NewMockClient(ctrl)
 	mockSFTPClient.EXPECT().Create(gomock.Any()).Return(&nopWriteCloser{remoteBuf}, nil)
+	mockSFTPClient.EXPECT().MkdirAll(gomock.Any()).Return(nil)
```

Comment:
would it be better if instead of using `gomock.Any()` actual value that is expected for the fn is used?

Reply:
Does it make any difference? Used Any() to increase the readability

Reply:
Using actual expected value, is better for testing point of view.